src/pkg/projects/Microsoft.NETCore.App.Ref/Microsoft.NETCore.App.Ref.pkgproj

```diff
@@ -39,4 +42,35 @@
     </ItemGroup>
   </Target>
 
+  <!--
+    Copy the files in the package's data/ dir to a layout directory. This is what the targeting pack
+    installer will place in the dotnet install dir.
+
+    This extracts from the nupkg (zip) directly. An alternative would be using the PackageFile
+    items, but there's some risk of handling TargetPath metadata differently than NuGet does. Using
+    the nupkg directly ensures results are identical.
+  -->
+  <Target Name="CreateTargetingPackLayout"
+          AfterTargets="CreatePackage">
+    <PropertyGroup>
+      <TargetingPackNupkgFile>$(PackageOutputPath)$(Id).$(PackageVersion).nupkg</TargetingPackNupkgFile>
+    </PropertyGroup>
+
+    <ZipFileGetEntries TargetArchive="$(TargetingPackNupkgFile)">
+      <Output TaskParameter="Entries" ItemName="TargetingPackNupkgEntries" />
+    </ZipFileGetEntries>
+
+    <ItemGroup>
+      <TargetingPackDataEntries
+        Include="@(TargetingPackNupkgEntries)"
+        Condition="$([System.String]::new('%(TargetingPackNupkgEntries.Identity)').StartsWith('data/'))" />
+    </ItemGroup>
+
+    <ZipFileExtractToDirectory
+      SourceArchive="$(TargetingPackNupkgFile)"
+      DestinationDirectory="$(DataLayoutDir)ref/$(Id)/$(Version)/"
```

Comment:
I count 3 to 1 on votes to keep packs/ over ref/

https://github.com/dotnet/designs/pull/50/files/b78838c38284d45f25798922dd6c01ca526df7dc#diff-ebf265d0476041dd8cab82fa24ab5b0e

Counting @dsplaisted on the 1 side for his email that said to use ref/.  I otherwise don't recall anyone having a strong opinion to change packs/

I think ref/ is inaccurate and also reads redundantly with .Ref in path.

Reply:
I'm happy with `packs/` too.